dbms/src/Storages/KVStore/MultiRaft/PrehandleSnapshot.cpp

```diff
@@ -189,7 +181,10 @@ static inline std::tuple<ReadFromStreamResult, PrehandleResult> executeTransform
         if (stream->isAbort())
         {
             stream->cancel();
-            res = ReadFromStreamResult{.error = ReadFromStreamError::Aborted, .extra_msg = "", .region = new_region};
+            res = ReadFromStreamResult{
```

Comment:
When abort, we will use abort_error, which is by default Aborted, but sometimes could be other value.